src/angular/Views/search-results.html

```diff
@@ -18,35 +18,38 @@
         </div>
         <div class="row">
           <h4 class="x-search-result-header">{{ 'SEARCH_RESULTS' | translate }}</h4>
-          <ul class="list-group">
+          <ul class="list-group list-group-padding">
 
             <div ng-repeat="result in results">
               <a class="list-group-item" ng-click="selectService(result.id);">
                 
                 <!-- location name section -->
-                <div class="row media-body">
-                  <h2 class="media-heading">{{result.properties.locationName}}: {{result.properties.activityName}}</h2>
-                  <img class="media-object pull-right" ng-src="{{getPartnerLogoUrl(result)}}" onError="this.onerror=null;this.style.display='none'">
+                <div class="list-result-wrapper">
+                  <div class="list-result-title">
+                    <h1>{{result.properties.locationName}}</h1>
+                    <p> {{result.properties.activityName}}</p>
+                  </div>
+                  <img class="list-result-image" ng-src="{{getPartnerLogoUrl(result)}}" onError="this.onerror=null;this.style.display='none'">
                 </div>
 
                 <div class="row">
 
                   <!-- hours section -->
                   <div class="col-xs-5">
-                    <h3>{{ 'HOURS' | translate }}</h3>
+                    <h2>{{ 'Hours' | translate }}</h2>
```

Comment:
You can't edit these keys like that. The key `HOURS` corresponds to the english and arabic translations in `translations.config.js`. If you want to change what it's showing in english, you need to edit the value of that key.

If you change these keys without changing the corresponding translations files then the string won't be translated.